### PR TITLE
controls for page structure and overlays

### PR DESCRIPTION
**Pull request by nataliepina**
Converting stories under Page Structure (Note: Fullscreen View will not be added due to the lack of need for controls there) and under Overlay.

## Testing

Test out stories under related folders. 

## Checklist for PR creator

- [ ] If any new components were added, there are exported from `packages/index.ts`
- [ ] If this PR is associated with a JIRA, it is mentioned in commit message footer ("Closes …")
- [ ] If this PR contains breaking changes, is stated in commit message body ("BREAKING CHANGE: …")
- [x] Info for applicable sections above is provided

### CODE REVIEW

File: packages/modal/stories/Modal.stories.tsx [hunk 1]
```diff
@@ -157,14 +203,15 @@ export const _LargeDialogModalWithFooter = () => (
...
+        {...args}
       >
         <ModalContent />
       </LargeDialogModalWithFooter>
     )}
   </ModalStoryContainer>
 );
 
-export const _FullscreenModal = () => (
+export const _FullscreenModal = args => (
```
Comment (seialkali) [inline packages/modal/stories/Modal.stories.tsx:214]: ❓ How come some of these story names start with an underscore and some don't?

File: packages/popover/popover.stories.tsx [hunk 2]
```diff
@@ -1,85 +1,64 @@
...
+  decorators: [
+    Story => (
+      <Box textAlign="center">
+        <Story />
+      </Box>
+    )
+  ],
+  component: Popover,
+  argTypes: {
+    preferredDirections: {
```
Comment (seialkali) [inline packages/popover/popover.stories.tsx:29]: I'm getting a console error when selecting a preferredDirections option in these stories as well:

`Uncaught (in promise) TypeError: Cannot read properties of undefined (reading 'x')`

File: packages/tooltip/stories/Tooltip.stories.tsx [hunk 3]
```diff
@@ -107,19 +109,14 @@ export const MinOrMaxWidth = () => (
   </>
 );
 
-export const ContentWidth = () => (
+export const ContentWidth = args => (
```
Comment (seialkali) [inline packages/tooltip/stories/Tooltip.stories.tsx:112]: I'm getting a console error here as well when trying to change the `preferredDirection` option on this story. The rest of the stories in this component seem to be fine though:

`Uncaught (in promise) TypeError: Cannot read properties of undefined (reading 'y')`

**Review by seialkali — COMMENTED** [review]
Overall these changes look good. There's just the console errors I noticed on a few of the stories when selecting a `preferredDirection`. I'm not sure if this is just an issue with storybook, or if it also affects the component in the package. Have a look at the errors and let me know what you think. If you feel like this can be merged without having to address the console errors first and coming back to it later, then I'm happy with that.

File: packages/modal/stories/Modal.stories.tsx (hunk 1 above)
Comment (nataliepina) [reply packages/modal/stories/Modal.stories.tsx:214]: They'll start with an underscore if they are the name of the component used. It renders in Storybook without the underscore.

File: packages/dropdownMenu/stories/Dropdown.stories.tsx [hunk 4]
```diff
@@ -19,11 +30,31 @@ export default {
     DropdownMenuItem,
     DropdownMenuItemIcon,
     DropdownMenuItemAvatar
+  },
+  argTypes: {
+    preferredDirections: {
```
[the remark this answers is not in the document]
Comment (nataliepina) [reply packages/dropdownMenu/stories/Dropdown.stories.tsx:35]: Will look into this.

File: packages/modal/stories/Modal.stories.tsx (hunk 1 above)
Comment (seialkali) [reply packages/modal/stories/Modal.stories.tsx:214]: Ahh ok, good to know, thanks!

File: packages/dropdownMenu/stories/Dropdown.stories.tsx [hunk 5]
```diff
@@ -41,8 +72,13 @@ export const Default = () => (
   </DropdownMenu>
 );
 
-export const WithDangerousAction = () => (
-  <DropdownMenu trigger={<PrimaryDropdownButton>Menu</PrimaryDropdownButton>}>
+export const Default = Template.bind({});
```
Comment (clintonmedbery) [inline packages/dropdownMenu/stories/Dropdown.stories.tsx:75]: ❓ - I am getting some accessibility errors. Is this a priority for us? 
image

**Review by clintonmedbery — DISMISSED** [review]
Nice work!

File: packages/dropdownMenu/stories/Dropdown.stories.tsx (hunk 5 above)
Comment (nataliepina) [reply packages/dropdownMenu/stories/Dropdown.stories.tsx:75]: Ah, good catch, this is pre-existing. I will log a separate ticket to fix the a11y.

**Review by vacas5 — CHANGES_REQUESTED** [review]
I would use a `chore` or a `refactor` prefix for your commit message, if nothing about the underlying components is changing, just the stories themselves. Same is true for all of these PRs, sorry. You should probably amend your commit messages.

File: packages/dropdownable/stories/helpers/DropdownStory.tsx [hunk 6]
```diff
@@ -1,64 +1,52 @@
...
+import {
+  themeBgPrimary,
+  themeBorder
+} from "../../../design-tokens/build/js/designTokens";
+
+export const DropdownContentContainer = styled.div`
+  min-width: "250px";
+  border: "1px solid ${themeBorder}";
+  background-color: ${themeBgPrimary};
+  padding: "5px";
```
Comment (seialkali) [inline packages/dropdownable/stories/helpers/DropdownStory.tsx:16]: The double quotes are breaking the styling. They're not needed here.

**Review by seialkali — APPROVED** [review]
Looks good!

**Review by vacas5 — APPROVED** [review]

**Pull request merged**